migrate/20231117_spdk_upgrade.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:tool, %w[spdk cloud-hypervisor])
+
+    # Prog::InstallSpdk, Prog::InstallCloudhypervisor:
+    #    - Create an instance.
+    # Prog::Vm::Nexus:
+    #    - "allocation_weight" determines in what ratio of cases this should be used.
+    #    - If no instances, use some default. Useful for the transition period.
+    # Prog::RemoveSpdk, Prog::RemoveCloudhypervisor:
+    #    - Error-out if the only installation
+    #    - Set "allocation_weight" to 0, and wait until all VMs or storage volumes
+    #      using this tool are destroyed.
+    create_table(:vm_host_tool_installation) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
```

Comment:
Seems like not a ubid. Should it be a UBID? Though, we should probably have a generic/opaque ubid that doesn't require cluttering up the class-prefix map. the problem with random UUIDs is now they can be confused for ubids of other types depending on the first ten bits.

Reply:
made it a ubid